### PR TITLE
Add solidity-coverage

### PR DESCRIPTION
**Pull request by izqui**

### CODE REVIEW

File: .coveralls.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+repo_token: w3im2b4XK8VhKdMeLgKqbG90f4I93rgUF
```
Comment (readevalprint) [inline .coveralls.yml]: ??

File: truffle.js [hunk 2]
```diff
@@ -5,11 +5,22 @@ var HDWalletProvider = require('truffle-hdwallet-provider');
 
 const mnemonic = 'stumble story behind hurt patient ball whisper art swift tongue ice alien';
 
+let developmentProvider, ropstenProvider, kovanProvider = {}
+
+if (!process.env.SOLIDITY_COVERAGE){
```
Comment (readevalprint) [inline truffle.js]: Nice!

File: .coveralls.yml (hunk 1 above)
Comment (izqui) [inline .coveralls.yml]: the token for coveralls to identify it is our repo

File: truffle.js (hunk 2 above)
Comment (izqui) [inline truffle.js]: Yay!

**Pull request merged**